lib/pages/chat/chat.dart

```diff
@@ -303,13 +304,9 @@ class ChatController extends State<Chat>
   void initState() {
     keyboardVisibilityController.onChange.listen(_keyboardListener);
     scrollController.addListener(_updateScrollController);
+    widget.jumpToEventIdStream?.stream.listen(scrollToEventId);
```

Comment:
Please don't forget to cancel this `stream subscription`

Reply:
done